src/plugins/components/snack/snack.component.ts

```diff
@@ -49,12 +49,39 @@ export const Snack = ({
       <span class="nui-snack-text">
         <slot>${label}</slot>
       </span>
-      ${ButtonClose({
-        shape: 'full',
-        classes: {
-          wrapper: 'nui-snack-button scale-75',
-        },
-      })}
+      ${size === 'xs'
+        ? html`
+            ${ButtonClose({
+              rounded: 'full',
+              size: 'xs',
+              classes: {
+                wrapper: 'nui-snack-button scale-75',
+              },
+            })}
+          `
+        : ''}
+      ${size === 'sm'
+        ? html`
+            ${ButtonClose({
+              rounded: 'full',
+              size: 'sm',
+              classes: {
+                wrapper: 'nui-snack-button scale-75',
+              },
+            })}
+          `
+        : ''}
+      ${size === 'md'
+        ? html`
+            ${ButtonClose({
+              rounded: 'full',
+              size: 'md',
+              classes: {
+                wrapper: 'nui-snack-button scale-75',
+              },
+            })}
+          `
+        : ''}
```

Comment:
```suggestion
		${ButtonClose({
	    size,
	    shape: 'full',
	    classes: {
	      wrapper: 'nui-snack-button scale-75',
	    },
	  })}
```

Reply:
Made the changes accordingly. Note: the shape prop does not exist anymore. Replaced by the rounded prop.